lib/airports.rb

```diff
@@ -12,6 +12,19 @@ def self.find_by_iata_code(iata_code)
     Airport.
       new(airport_data.each_with_object({}) { |(k, v), h| h[k.to_sym] = v })
   end
+  
+  def self.find_by_city(city)
```

Comment:
Is this method useful? What about where a city has multiple airports? Would you want the one that *just so happens* to be first in the dataset?